client/blocks/product-purchase-features-list/video-audio-posts.jsx

```diff
@@ -30,7 +31,7 @@ function getDescription( plan, translate ) {
 				'directly to your site — the Pro Plan has 50 GB storage.'
 		);
 	}
-	if ( isWpComEcommercePlan( plan ) ) {
+	if ( isWpComEcommercePlan( plan ) || isWooExpressPlan( plan ) ) {
```

Comment:
The Woo Express Essential plan only includes 50GB of storage,

Reply:
Hey Dale! I addressed the problem you mentioned here: #76515. I just added this condition here in case this PR gets merged before #76515.